### PR TITLE
Mention future minset additions

### LINKED ISSUE #1081: Discussion about impact on the architecture when the minset expands
https://github.com/ietf-tapswg/api-drafts/blob/9266fbe3b55966edaf4013e755d46ae6f0fed8de/draft-ietf-taps-arch.md?plain=1#L226

Assuming there would be more protocols in the future and the minset functionalities will expand via extension or amendments. It would important to indicate if this architecture should is also extensible or it only supports on the minimum set referred. Definitely we haven't seen the future yet, but it would be important to express if we have considered such expansion of minimum functionalities.

### PR DESCRIPTION
**Pull request by tfpauly**
Closes #1081

### CODE REVIEW

File: draft-ietf-taps-arch.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ The normative requirements described in this section allow Transport Services AP
 
 ## Provide Common APIs for Common Features
 
-Any functionality that is common across multiple transport protocols SHOULD be made accessible through a unified set of calls using the Transport Services API. As a baseline, any Transport Services API SHOULD allow access to the minimal set of features offered by transport protocols {{?RFC8923}}.
+Any functionality that is common across multiple transport protocols SHOULD be made accessible through a unified set of calls using the Transport Services API. As a baseline, any Transport Services API SHOULD allow access to the minimal set of features offered by transport protocols {{?RFC8923}}. If that minimal set is updated and expanded in the future, the Transport Services API ought to be extended to match, if necessary.
```
Comment (mwelzl) [inline draft-ietf-taps-arch.md:226]:
```suggestion
Any functionality that is common across multiple transport protocols SHOULD be made accessible through a unified set of calls using the Transport Services API. As a baseline, any Transport Services API SHOULD allow access to the minimal set of features offered by transport protocols {{?RFC8923}}. If that minimal set is updated and expanded in the future, the Transport Services API ought to be extended to match.
```

I just find the "if necessary"  a bit too much ... "if it happens, it ought to, if necessary". I'm not even sure what "if necessary" means here... when would doing that become necessary, and when not? What would make it necessary - compatibility, business reasons?

**Review by mwelzl — APPROVED** [review]
LGTM modulo a nit

File: draft-ietf-taps-arch.md (hunk 1 above)
Comment (gorryfair) [reply draft-ietf-taps-arch.md:226]: also: I think it's /updated **or** extended/ not both.

Comment (tfpauly) [reply draft-ietf-taps-arch.md:226]:
```suggestion
Any functionality that is common across multiple transport protocols SHOULD be made accessible through a unified set of calls using the Transport Services API. As a baseline, any Transport Services API SHOULD allow access to the minimal set of features offered by transport protocols {{?RFC8923}}. If that minimal set is updated or expanded in the future, the Transport Services API ought to be extended to match.
```

Comment (tfpauly) [reply draft-ietf-taps-arch.md:226]: I originally meant "if necessary" to say that the minset change might not require an API change, but I can remove.

**Pull request merged**